rpc-rs/Cargo.toml

```diff
@@ -45,6 +48,7 @@ bigdecimal = { version = "0.3", optional = true }
 tokio = { version = "1", features = ["full"]}
 futures = "0.3"
 nats-aflowt = "0.16.104"
+nats = { version = "0.17", features=["unstable"], optional=true }
```

Comment:
even though their documentation says it requires the unstable feature, you don't actually have to include it.

Reply:
removed